src/lib/websocket/index.ts

```diff
@@ -0,0 +1,293 @@
+import WebSocket, { Server, Data } from 'ws';
```

Comment:
I'd prefer if this file be split into multiple smaller files so it's not 294 lines of blurb. Move the already exported interfaces and enums to `constants.ts` or `interfaces.ts`, the class `WebsocketHandler` to `Handler.ts` and the class `DashboardWebsocketUser` to `DashboardUser.ts`.